dataland-frontend/src/components/forms/CreateLksgDataset.vue

```diff
@@ -186,6 +197,18 @@ export default defineComponent({
         // IGNORED
       },
     },
+    isSubcategoryVisible(): Map<Subcategory, boolean> {
+      const map = new Map<Subcategory, boolean>();
+      for (const category of this.lksgDataModel) {
+        for (const subcategory of category.subcategories) {
+          map.set(
+            subcategory,
+            subcategory.fields.some((field) => field.showIf(this.companyAssociatedLksgData.data))
+          );
+        }
+      }
+      return map;
+    },
```

Comment:
The name `isSubcategoryVisible()` implies that this method returns the answer to that question via `true` or `false`, but it only builds the map and returns ist.   You have to do a `.get()` on the returned map and check if the value is `true`.

So why not doing that last check inside of the function itself?
So:

`isSubcategoryVisible(nameOfSubcategory: string): Boolean {`
  `...buildthemap...`
  `return isSubcategoryVisible.get(subcategory) ?? true `

Reply:
or rename the map subcategoryVisibility?

Reply:
Done. Change name to `subcategoryVisibility`.

Reply:
👍  that's fine for me